src/components/TopBar/TopBar.tsx

```diff
@@ -43,7 +46,13 @@ class TopBar extends React.Component <{
 	onChangeLanguage: (language: string, afterChange: () => Promise<void>) => void
 	darkMode: boolean
 	onChangeDarkMode: (dark: boolean) => void
+    multiShiftMode: boolean
+    onChangeMultiShiftMode: (multiShiftMode: boolean) => void
+    disableMultiShiftModeChange: boolean
```

Comment:
I believe that these lines aren't well aligned

Reply:
bah, the linter did nothing :sob:
fixed